src/content/developers/tutorials/how-to-view-nft-in-metamask.md

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Same with first two, make sure to add "frontmatter" metadata section up here (see other tutorials for examples)